src/azure-cli-core/azure/cli/core/commands/__init__.py

```diff
@@ -72,7 +72,8 @@ class CliArgumentType(object):
 
     def __init__(self, overrides=None, **kwargs):
         if isinstance(overrides, str):
-            raise ValueError("Overrides has to be a CliArgumentType (cannot be a string)")
+            raise ValueError(
+                "Overrides has to be a CliArgumentType (cannot be a string)")
```

Comment:
Oh! The formatting must have been changed by the linting extension I've on VSCode. Will fix this